packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -61,6 +61,22 @@ The maximum size of the uploaded file that is validated by the JavaScript plugin
 </div>
 ```
 
+### Dismissible Drop Zone
```

Comment:
Please update this section accordingly once we resolve the prop name.

Reply:
done